docs/content/administration/command-line.en-us.md

```diff
@@ -95,6 +95,7 @@ Admin operations:
       - Options:
         - `--username value`, `-u value`: Username. Required.
         - `--password value`, `-p value`: New password. Required.
+        - `--must-change-password`: If provided, the user is required to choose a new password after the login. Optional. (default: true).
```

Comment:
```suggestion
        - `--must-change-password`, `--must-change-password=true/false`: If provided, the user is required to choose a new password after the login. Optional.
```

The ` (default: true)` and `optional` might cause misunderstanding IMO.

I think everyone could understand that `--must-change-password` means `--must-change-password=true`

Reply:
Is there any other suggestion? If no, could I commit this suggestion and approve?

Reply:
The suggestion sounds good

Reply:
Changed the wording. Now the default is `false` if the flag is not set. I still think this should default to `true`.

Reply:
Sure, but I think if it defaults to true, then it might needs some new wording. (the old one makes me think for quite a long time to understand what is "optional" and what is "default: true" ....)

Maybe like this?


```

- `--must-change-password`: By default the user is required to choose a new password after login, use `--must-change-password=false` to disable this option.

```

Anyway, no better idea from my side. Either is fine to me.

Reply:
I think I just used the wording from here:
https://github.com/go-gitea/gitea/blob/360b3fd17c3315ad9ad9c4e6ac02eda73f48d8ae/docs/content/administration/command-line.en-us.md?plain=1#L86-L87

Maybe we should have a common format for optional and required parameters.